### PR TITLE
Remove linux wheel from circleci

### PR DESCRIPTION
**Pull request by atalman**
Remove linux wheel from circleci.
Since we removed linux wheel from circleci, we want to make sure regenerate.py works correctly with the latest changes

### CODE REVIEW

**Review by weiwangmeta — APPROVED** [review]
Looks good!

File: .circleci/regenerate.py [hunk 1]
```diff
@@ -55,7 +58,7 @@ def build_workflows(prefix="", upload=False, filter_branch=None, indentation=6):
         # Build on every pull request, but upload only on nightly and tags
         w += build_doc_job("/.*/")
         w += upload_doc_job("nightly")
-        w += docstring_parameters_sync_job(None)
+        # w += docstring_parameters_sync_job(None)
```
[the remark this answers is not in the document]
Comment (atalman) [reply .circleci/regenerate.py:61]: Hm I think this would be a question to @osalpekar . Was this job implemented ?

**Review by hwangjeff — APPROVED** [review]

File: .circleci/regenerate.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mthrok) [reply .circleci/regenerate.py:61]: @osalpekar Did you comment out this? This job cannot be removed. This checks if the docstring are correctly in-sync with actual parameters. This job needs to be revived.

[the remark this answers is not in the document]
Comment (osalpekar) [reply .circleci/regenerate.py:61]: No, looks like this this was commented out in the current PR. We can re-add it safely AFAICT

[the remark this answers is not in the document]
Comment (atalman) [reply .circleci/regenerate.py:61]: @mthrok I think this job was disabled by this PR: https://github.com/pytorch/audio/pull/2702 
Just commented on it

**Pull request closed without merging**